Added particle f2(1270) in the O2Database accroding to PDG 2024

sawankumawat opened the pull request:
-- Particle f2(1270) was added for the glueball study in the KsKs mass spectrum.
-- The mass and width are taken from the PDG 2024.
-- BR is set to 100% for the decay mode to KsKs channel, which will be the channel used in the study.

sawankumawat commented:
+async-label async-2023-pbpb-apass4, async-2022-pp-apass7, async-2024-pp-apass1, async-2024-PbPb-apass1

sawenzel reviewed (changes requested):
Most lines changed have nothing to do with this PR. Please provide a minimal set of changes in this commit (no whitespace changes or formatting changes).

sawankumawat committed:
added particle f2(1270)

sawankumawat commented:
> Most lines changed have nothing to do with this PR. Please provide a minimal set of changes in this commit (no whitespace changes or formatting changes).

Dear @sawenzel, thanks for reviewing the pull request. The white space changes were automatically done be the VS code extension. I have now changed it and just added the particle f2(1270) in the O2Database.

alibuild commented:
Error while checking build/O2/fullCI_slc9 for 83a4df1c2049ebe26a9571d01e994f3cfc4abe49 at 2025-04-07 06:31:
```
## sw/BUILD/O2Physics-latest/log
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:


## sw/BUILD/O2-full-system-test-latest/log
Detected critical problem in logfile digi.log
digi.log:[20446:internal-dpl-ccdb-backend]: [06:31:43][ERROR] Exception while running: Fatal error. Rethrowing.
digi.log-[20446:internal-dpl-ccdb-backend]: [06:31:43][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-sim-digitizer-workflow, device shutting down. Reason: Fatal error
[20446:internal-dpl-ccdb-backend]: [06:31:41][ERROR] CCDBDownloader CURL transfer error - Timeout was reached
[20446:internal-dpl-ccdb-backend]: [06:31:41][ERROR] CcdbDownloader finished transfer http://alice-ccdb.cern.ch/CTP/Calib/OrbitReset for 1550600800000 (agent_id: alimetal04.cern.ch-1744000294-rCCqkj) with http code: 0
[20446:internal-dpl-ccdb-backend]: [06:31:41][ERROR] File CTP/Calib/OrbitReset could not be retrieved. No more hosts to try.
[20446:internal-dpl-ccdb-backend]: [06:31:41][FATAL] Unable to find CCDB object CTP/Calib/OrbitReset/1550600800000
[20446:internal-dpl-ccdb-backend]: [06:31:43][ERROR] Exception while running: Fatal error. Rethrowing.
[20446:internal-dpl-ccdb-backend]: [06:31:43][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-sim-digitizer-workflow, device shutting down. Reason: Fatal error
[ERROR] Workflow crashed - PID 20446 (internal-dpl-ccdb-backend) did not exit correctly however it's not clear why. Exit code forced to 128.


## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
++ GRERR=1
++ [[ 1 == 0 ]]
++ mkdir -p /sw/INSTALLROOT/503dc91cebe8fcd055bcbd23d2a00246ff58e99d/slc9_x86-64/o2checkcode/1.0-local112/etc/modulefiles
++ cat
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13928/83a4df1c2049ebe26a9571d01e994f3cfc4abe49/build_O2_fullCI_slc9/pretty.html).

alibuild commented:
Error while checking build/O2/fullCI for 83a4df1c2049ebe26a9571d01e994f3cfc4abe49 at 2025-02-11 12:04:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/O2Physics-latest/log
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:


## sw/BUILD/O2-sim-challenge-test-latest/log
./sim-challenge.logDetected critical problem in logfile digi.log
./sim-challenge.logdigi.log:[8963:internal-dpl-ccdb-backend]: [11:04:44][ERROR] Exception while running: Fatal error. Rethrowing.
./sim-challenge.logdigi.log-[8963:internal-dpl-ccdb-backend]: [11:04:44][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-sim-digitizer-workflow, device shutting down. Reason: Fatal error
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/37}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/38}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/40}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/42}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/43}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/45}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/46}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/47}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/48}
./sim-challenge.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/49}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/37}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/38}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/40}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/42}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/43}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/45}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/46}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/47}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/48}
./sim-challenge.log[8961:internal-dpl-clock]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/49}
./sim-challenge.log[8978:TPCDigitizer_13]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/37}
./sim-challenge.log[8978:TPCDigitizer_13]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/38}
./sim-challenge.log[8978:TPCDigitizer_13]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/40}
./sim-challenge.log[8978:TPCDigitizer_13]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/42}
./sim-challenge.log[8978:TPCDigitizer_13]: [ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/43}
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13928/83a4df1c2049ebe26a9571d01e994f3cfc4abe49/build_O2_fullCI/pretty.html).
